obsidian.css

```diff
@@ -487,3 +487,9 @@ input.task-list-item-checkbox:checked {
     fill: var(--frost3) !important;
 }
 
+.setting-item-control input[type="text"] {
+    color: var(--text-normal);
```

Comment:
I decided to include this line too even if it doesn't change color at all (`--text-normal` is default for such inputs). Why? Because I wanted to make it explicitly visible that there is some intentionally chosen color for an input value, then we change it to something different for a placeholder 3 lines below here.